x-pack/legacy/plugins/monitoring/public/components/no_data/explanations/exporters/exporters.js

```diff
@@ -72,3 +72,20 @@ ExplainExporters.propTypes = {
   property: PropTypes.string.isRequired,
   data: PropTypes.string.isRequired
 };
+
+export function ExplainExportersCloud() {
+  return (
+    <Fragment>
+      <CloudDeployment />
+      <EuiHorizontalRule size="half" />
+      <EuiText className="eui-textLeft">
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.noData.explanations.exportersCloudDescription"
+            defaultMessage="In Elastic Cloud, your monitoring data is stored in your dedicated monitoring cluster."
```

Comment:
Is there some documentation link we can add here? It feels a bit bare in it's current form.